doc/install.rst

```diff
@@ -0,0 +1,43 @@
+==========
+Installing
+==========
+
+.. raw:: html
```

Comment:
Why are we not using bootstrap to do this?

Reply:
Not sure there is a similar feature in bootstrap. [scikit-learn also uses it](https://raw.githubusercontent.com/scikit-learn/scikit-learn/main/doc/install.rst) so I've stuck with it :)
Although, we could use bootstrap elements instead of the CSS provided by quickstart. I haven't managed to get it working yet, but I'll investigate that afterwards!